pkg/tests/integration/test_salt_ufw.py

```diff
@@ -0,0 +1,30 @@
+import logging
+import os.path
+import pathlib
+import subprocess
+
+import pytest
+from pytestskipmarkers.utils import platform
+
+log = logging.getLogger(__name__)
+
+
+@pytest.mark.skip_on_windows()
+def test_salt_ufw(salt_master, install_salt):
+    """
+    Test salt.ufw for Debian/Ubuntu salt-master
+    """
+    log.warning(f"DGM test_salt_ufw install_salt '{install_salt}'")
+
+    if install_salt.distro_id not in ("debian", "ubuntu"):
+        pytest.skip("Only tests Debian / Ubuntu packages")
+
+    pkg = [x for x in install_salt.pkgs if "deb" in x]
+    if not pkg:
+        pytest.skip("Not testing deb packages")
```

Comment:
```suggestion
        pytest.skip("No deb packages to test")
```
?

Reply:
I should have added WIP, wanted to get the work saved off while addressed a different issue, and built so that I can try the new package testing instructions. So disregard this test as it will change. Btw: use warning log level for debug stuff since then don't have to bother with editing config file for minion